common/src/main/java/io/github/axolotlclient/modules/hypixel/bedwars/BedwarsPlayerStats.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright © 2021-2023 moehreag <moehreag@gmail.com> & Contributors
+ *
+ * This file is part of AxolotlClient.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * For more information, see the LICENSE file.
+ */
+
+package io.github.axolotlclient.modules.hypixel.bedwars;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import io.github.axolotlclient.modules.hypixel.HypixelAbstractionLayer;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * @author DarkKronicle
+ */
+
+@AllArgsConstructor
+public class BedwarsPlayerStats {
+
+	@Getter
+	private int finalKills;
+	@Getter
+	private int finalDeaths;
+	@Getter
+	private int bedsBroken;
+	@Getter
+	private int deaths;
+	@Getter
+	private int kills;
+	@Getter
+	private int gameFinalKills;
+	@Getter
+	private int gameFinalDeaths;
+	@Getter
+	private int gameBedsBroken;
+	@Getter
+	private int gameDeaths;
+	@Getter
+	private int gameKills;
+	@Getter
+	private final int losses;
+	@Getter
+	private final int wins;
+	@Getter
+	private final int winstreak;
+	@Getter
+	private final int stars;
+
+
+	public static BedwarsPlayerStats generateFake() {
+		return new BedwarsPlayerStats(
+			0,
+			0,
+			0,
+			0,
+			0,
+			0,
+			0,
+			0,
+			0,
+			0,
+			0,
+			0,
+			0,
+			1
+		);
```

Comment:
```suggestion
		return new BedwarsPlayerStats(0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1);
```

Reply:
done!

Reply:
To be compliant with hypixel API terms this needs to generate fake stats so it can't be used to test if someone has a nick. I can come up with the algorithm

Reply:
okay!